### PR TITLE
Icons: Remove aria-hidden to fix narrator focus bug

### PR DESCRIPTION
**Pull request by joschect**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Icons: Remove aria-hidden to fix narrator focus bug

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by cschlechty** [comment]
This addresses an issue where narrator focus highlight doesn't work as expected. While narrator should address that bug, I believe this is the right behavior for icons. Present to screen readers, but presentational.

File: packages/utilities/src/styled.tsx [hunk 1]
```diff
@@ -41,7 +41,8 @@ const DefaultFields = ['theme', 'styles'];
 export function styled<
   TComponentProps extends IPropsWithStyles<TStyleProps, TStyleSet>,
   TStyleProps,
-  TStyleSet extends IStyleSet<TStyleSet>
+  TStyleSet extends IStyleSet<TStyleSet>,
```
Comment (micahgodbolt) [inline packages/utilities/src/styled.tsx]: is this part of this PR?

**Comment by micahgodbolt** [comment]
OMG this is helpful 
image

File: packages/utilities/src/styled.tsx (hunk 1 above)
Comment (joschect) [reply packages/utilities/src/styled.tsx]: Ack, no.

**Comment by dzearing** [comment]
Run update-snapshots in experiments.

**Comment by dzearing** [comment]
Ping @joschect 

I might enlist to help you out here, this is getting pretty stale.

**Comment by dzearing** [comment]
@joschect helping you out here.

**Pull request merged**